test/unit/testTwilioHelpers/testBuyAndConfigureTwilioPhoneNumber.js

```diff
@@ -0,0 +1,183 @@
+// Third-party dependencies
+const { expect } = require('chai')
+const { afterEach, beforeEach, describe, it } = require('mocha')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+const chai = require('chai')
+const rewire = require('rewire')
+
+// In-house dependencies
+const helpers = require('../../../lib/helpers')
+
+const twilioHelpers = rewire('../../../lib/twilioHelpers')
```

Comment:
Rewire allows me to mock a function that is internal to `twilioHelpers.js` (i.e. it's not included in the `module.exports`. This is what is happening with the `__set__` functions a little later on in this file.